packages/providers/src/core/PoapTokenApi/PoapTokenApi.ts

```diff
@@ -0,0 +1,98 @@
+import {
+  CheckCodeResponse,
+  ClaimCodeResponse,
+  ClaimStatusResponse,
+} from './../../ports/TokensApiProvider/Types/response';
+import { ClaimCodeInput } from './../../ports/TokensApiProvider/Types/input';
+import { AuthenticationProvider } from './../../ports/AuthenticationProvider/AuthenticationProvider';
+import { TokensApiProvider } from './../../ports/TokensApiProvider/TokensApiProvider';
+/* eslint-disable @typescript-eslint/no-explicit-any */
```

Comment:
This should be the first line of the file. Even better is you could use `unknown` instead of `any`.